release.wdl

```diff
@@ -0,0 +1,238 @@
+#Thoughts--
+#Still need to manually check that the jars are being created in broad artifactory
+#Need to manually investigate if tests aren't being written
+#Need to manualy update the release portion of Cromwell
+#Need to make sure that the changelog.md file doesn't get weirdly altered when merging to master for Cromwell
```

Comment:
markdown doesn't auto-merge well it turns out. we ran into it when we did our release and @katevoss just ran into it with the wdl spec